apps/hip/templates/includes/sidebar.html

```diff
@@ -114,7 +114,7 @@ <h5 class="is-size-7">Resources</h5>
       <li class="pl-4 pb-2 sidenav-link-hip" data-href="/posters/">Posters</li>
       <li class="pl-4 pb-2 sidenav-link-hip" data-href="/related-websites/">Related Websites</li>
       <li class="pl-4 pb-2 sidenav-link-hip" data-href="/conferences/">Conferences</li>
-      <li class="pl-4 pb-2 sidenav-link-hip" data-href="/newsletter/">Newsletter</li>
+      <li class="pl-4 pb-2 sidenav-link-hip" data-href="/newsletters/">Newsletters</li>
```

Comment:
We'll need to make sure to change the slug for the page on staging so that this doesn't 404.